src/main/java/com/algorand/algosdk/transaction/HeartbeatProof.java

```diff
@@ -0,0 +1,74 @@
+package com.algorand.algosdk.transaction;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import com.algorand.algosdk.crypto.*;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+
+@JsonPropertyOrder(alphabetic = true)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class HeartbeatProof implements Serializable {
+    @JsonProperty("s")
```

Comment:
I *think* each of these json properties need to match those returned from the indexer transactions endpoint. This one for example should be `hb-sig`:

```
{
    "current-round": 43895147,
    "transaction": {
        "close-rewards": 0,
        "closing-amount": 0,
        "confirmed-round": 43874103,
        "fee": 0,
        "first-valid": 43874101,
        "genesis-hash": "mFgazF+2uRS1tMiL9dsj01hJGySEmPN28B/TjjvpVW0=",
        "heartbeat-transaction": {
            "hb-address": "IMQCBCFVQXEAMKEP6Q35VGCTOTGFK2KWLGDGWY3RU3KEZGQ2VQP3STQ4CU",
            "hb-key-dilution": 448,
            "hb-proof": {
                "hb-pk": "dN+71r3ZdF73q/TGDM9xVuflvJ6NH7HRi5vPGLfm2W0=",
                "hb-pk1sig": "avYRYFGMNxEk2uqJltoBHBAtR5lueGxfIyMJE5TXK3m0fGMuzT0VJ326N5P9mXy7rn/H9LknS/ya3XlL0ixZCg==",
                "hb-pk2": "gqPeNVGldRTpCvFdBcL+L3ucpHy1ug7CSAWXBI2DEs8=",
                "hb-pk2sig": "uDbmE1/RvVdDbQH4mJdktYgpLEDfM991hMs2rQJKl9cxRF1wfJ8oxJ3G2P9ZBzbqOfgDb6Q9mBCjxBo+jhgRCg==",
                "hb-sig": "MJQXyK38FX8w4OU2lmH0Ckx2oTUdLO9a+T0yECd3kg8j82gMCHoE0bhXiHZU6Oe2gK5TIvDkcs9EmWF+KU0SBQ=="
            },
            "hb-seed": "GF0hM6bj9usJyLdG7Byj8Q1Q8DJHtFNamHN48qA4pXI=",
            "hb-vote-id": "W6ROFROYH51gd3A5njH9eHcTBfufAIwGp8IHptXAo84="
        },
        "id": "EVVXDK6Z3IU63NA6UFFYPR2WGVHFGZDTBLU3W3WZIBDAOYH3OTOQ",
        "intra-round-offset": 0,
        "last-valid": 43874111,
        "receiver-rewards": 0,
        "round-time": 1735174805,
        "sender": "GAU5WA6DT2EPFS6LKOA333BQP67NXIHZ7JPOOHMZWJDPZRL4XMHDDDUCKA",
        "sender-rewards": 0,
        "signature": {
            "logicsig": {
                "args": [],
                "logic": "CzEgMgMS"
            }
        },
        "tx-type": "hb"
    }
}
```

Reply:
I'm completely wrong about this; this is the encoding for algod/indexer's are code-generated (and covered in another PR.